Localide/Classes/Localide.swift

```diff
@@ -172,11 +174,13 @@ public final class Localide: NSObject {
         completion: LocalideUsageCompletion?
     ) {
         if app == .copilot {
-            if let urlString = customUrlsPerApp[.copilot] {
-                _ = LocalideMapApp.launchAppWithUrlString(urlString)
-            } else if let copilotWithoutParams = LocalideMapApp.prefixes[.copilot] {
-                _ = LocalideMapApp.launchAppWithUrlString(copilotWithoutParams)
+            var openStatus: Bool = false
+            if let urlString = customUrlsPerApp[app] {
+                openStatus = LocalideMapApp.launchAppWithUrlString(urlString)
+            } else if let copilotWithoutParams = LocalideMapApp.prefixes[app] {
+                openStatus = LocalideMapApp.launchAppWithUrlString(copilotWithoutParams)
             }
+            completion?(app, fromMemory, openStatus)
```

Comment:
Can we unit test this?

unit tests should cover all known apps and verify the completion block is called in all cases

Reply:
opened jira to do the required changes so we can mock the process: https://bringg.atlassian.net/browse/BRNGG-29255